projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -90,11 +91,13 @@ public function enqueue_admin_scripts() {
 				'textdomain' => 'jetpack-social',
 			)
 		);
+
+		global $publicize;
+		wp_localize_script( 'jetpack-social', 'connections', $publicize->get_all_connections_for_user() );
```

Comment:
You probably want to add this as a namespaced item with an array of items, to avoid adding too many global variables to the JS. You could do something like this:

```php
wp_localize_script(
	'jetpack-social',
	'jetpackSocial',
	array(
		'connections' => $publicize->get_all_connections_for_user(),
	)
);
```

Then, in the JS you could access it like this: `jetpackSocial.connections`. This prevents naming collisions and makes it easier to add extra items down the road.